linux_os/guide/system/var_blacklisted_paths.var

```diff
@@ -0,0 +1,15 @@
+documentation_complete: true
+
+title: 'Blacklisted paths'
```

Comment:
I assume you want multiple rules sharing this variable. Why do you think so? Shouldn't each rule have a different variable?

Reply:
That's up to a discussion. My idea was to provide a way how to reliably disable a folder one is 100% sure that it shouldn't be ever scanned (dynamic FS, network FS).

Reply:
Sounds great!

Let me add some arguments against it. I think is that people won't know which all rules are affected by this variable, so they might think it's only one or all of them. Also, unless you change all the rules you won't make sure that the scanner won't access the file system.

But I think that your proposal is better in terms of user experience, as people will tailor only a single variable. Also, it will be easier approach for the content developers.

@yuumasato @vojtapolasek Any other ideas?

Reply:
I would prefer to avoid having one general variable to skip paths, different rules may want skip a path for different reasons.

For this case, I see that intention is to list all the remote paths.
The actual skipping or including happens at the rule's check, so I'd just name the variable `var_remote_paths`, (or filesystems), and describe it as an aggregator of paths in the system known to be pointing to remote file system.
If there would be rules dedicated to checking just remote filesystems the same variable can be used.

Reply:
I also would prefer to have this variable applied only to a set of rules where it really makes sense to apply. If one wants to skip a path for every rule that would make sense to be implemented on the scanner side, as IIRC OpenSCAP already does that through environment variables. But that still leaves us with the problem described in another comment: https://github.com/ComplianceAsCode/content/pull/5845#issuecomment-652530450

Reply:
Although I am not aware of a possibility to exclude a directory tree by "talking" directly to the scanner. Conversely, we are unable to guarantee that the whole set of applicable rules will get this functionality, so they can be influenced by this variable.
On the other hand, if you have a problem that a network directory is scanned, and you would like to prevent it, having to configure many variables in the same way seems to be user-unfriendly.

I see two options:
1. Multiple (probably < 10) variables for rules that scan the whole local filesystem by default.
2. One variable that is used by rules that are affected by the previous point as well.